content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -503,55 +503,99 @@ As of today, 7 fields are supported in the rules engine, and each of them has it
 The possible operators for the `created` field are:
 - =
 - !=
-- “>”
+- \>
```

Comment:
nope, `>` has a special meaning in yaml (multiline input), that's why it was surrounded by quotes

Reply:
ah!! because I was told that it was strange to have "" only for >